hpack/table.py

```diff
@@ -105,6 +111,13 @@ class HeaderTable(object):
 
     STATIC_TABLE_LENGTH = len(STATIC_TABLE)
 
+    STATIC_TABLE_MAPPING = {}
+    for index, (name, value) in enumerate(STATIC_TABLE):
+        header_name_search_result = STATIC_TABLE_MAPPING.setdefault(name, _HeaderNameSearchResult())
+        header_name_search_result.positions[value] = index
+        min_index = header_name_search_result.min
+        header_name_search_result.min = min(min_index, index) if min_index is not None else index
```

Comment:
Move to staticmethod, IMHO more convenient way is use metaclass at this case, but for python2/3 compatibility metaclass require some magic like `six.with_metaclass`. I can rewrite if the current choise does not suit.